pyproject.toml

```diff
@@ -83,6 +81,8 @@ grpcio = "1.68.1"
 protobuf = "5.29.2"
 pynacl = "^1.5.0"
 tqdm = "^4.67.1"
+frozenlist = "^1.4.0"
+multidict = "^6.0.5"
```

Comment:
what are these?

Reply:
this should be in my projects' dependencies, my bad, I was trying to make driftpy compatible with another package, they both use (frozendict and multidict, so pinning the version was needed). I have removed it.